data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,604 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	"<first>! I found a company who's willing to publish me! I sent them a digital copy of my book and they instructed me to visit their office on <planet>," explains Anna, once again enthusiastic. At a table behind her you notice the figure of Temporo, speaking to a waiter.`
+			choice
+				`	"Great!"`
+					goto go
+				`	"Let's go!"`
+					goto go
+				`	"What's the publishing company called?"`
+			`	"They're called 'Priyepichia Publishing.'" Anna shows you beautiful pictures of a clean white building perched on a mountainside, high above gentle waves. On the roof stands a landing pad painted with a blue dolphin.`
+			label go
+			`	You head back to your ship with Anna and Temporo, and plot a course to <destination>.`
+				accept
+	on complete
+		event "beyond the hyperdrive is published" 165 197
+		conversation
+			`Landing on <planet>, Anna is excitedly holding a copy of 'Beyond the Hyperdrive.' She hastily opens the hatch as soon as it unlocks.`
+			`	Anna stops, waiting for you to catch up. "In the message they sent me, they included instructions on how to reach their offices," she explains. "Let's go!"`
+			`	Several long streets later, Anna is completely lost. "This isn't where the instructions were supposed to lead at all," she says, turning to Temporo, who has been following you the whole time. "Do you know how to get there?"`
+			`	The Heliarch takes the lead, and within minutes you reach a train station. Temporo leads you to platform 61A. When the train arrives, you board it and travel to a neighborhood where a large white building dominates the skyline.`
+			`	Once the train stops, you get off onto the platform and exit the station. Circling around to the publisher's office, you find the entrance bustling with Kimek, along with the occasional Saryd and Arach coming and going. Anna looks for a front desk and directs you to a Kimek who appears to be a receptionist.`
+			choice
+				`	"Hello, do you happen to know where Priyepichia Publishing is?"`
+				`	"We're looking for a publishing company called Priyepichia Publishing. Could you provide directions?"`
+			`	"On the 16th floor, Priyepichia Publishing is. Over there, the elevators are," says the receptionist's translator.`
+			`	You turn to where the Kimek points you and see six large elevators, larger than what you are familiar with in human space. Temporo leads you to the least crowded one.`
+			`	The three of you enter the elevator. It makes stops at each floor of the building, more and more Kimek reach their destinations, and the elevator gets more and more quiet. When the door opens on the 16th floor, the elevator is only half full. Before you stands a Kimek who boarded at the eighth floor, wearing an ornate jeweled necklace as well as a translation device.`
```

Comment:
Kind of odd that the publishing company would have their greeter get on the elevator with you on a random floor and not do anything. Probably makes more sense for the greeter to be waiting on the publisher's floor instead.
```suggestion
			`	The three of you enter the elevator. It makes stops at each floor of the building, more and more Kimek reach their destinations, and the elevator gets more and more quiet. When you reach the 16th floor, the elevator is only half full. The door opens to reveal a Kimek, wearing an ornate jeweled necklace as well as a translation device.`
```